docs/about/themes-translation.md

```diff
@@ -0,0 +1,195 @@
+extracting messages from mkdocs/themes/readthedocs/versions.html (ignore_tags="script,style", include_attrs="alt title summary")
+writing PO template file to mkdocs/themes/readthedocs/messages.pot
+```
+
+#### Initializing a localization catalog
+
+Now that the text to be translated is extracted, create a catalog to host its
+translation for the target locale:
+
+```bash
+$ python setup.py init_catalog -t mkdocs -l es
+running init_catalog
+creating catalog mkdocs/themes/mkdocs/locales/es/LC_MESSAGES/messages.po based on mkdocs/themes/mkdocs/messages.pot
+
+$ python setup.py init_catalog -t readthedocs -l es
+running init_catalog
+creating catalog mkdocs/themes/readthedocs/locales/es/LC_MESSAGES/messages.po based on mkdocs/themes/readthedocs/messages.pot
+```
+
+Which will create:
+
+```text
+mkdocs/themes/mkdocs/locales
+├── es
+│   └── LC_MESSAGES
+│       └── messages.po
+...
+mkdocs/themes/readthedocs/locales
+├── es
+│   └── LC_MESSAGES
+│       └── messages.po
+```
+
+### Translating the MkDocs themes
+
+Now that your localized `messages.po` files are ready, all you need to do is
+add a translation in each `msgstr` item for each `msgid` item in the file.
+
+```text
+msgid "Next"
+msgstr "Siguiente"
+```
+
+!!! Warning
+    Do not modify the `msgid` as it is common to all translations. Just add
+    its translation in the `msgstr` item.
+
+### Testing themes translations
+
+Once you're done translating you'll want to test your localized theme.
+
+For this you need to compile the `messages.po` files into `messages.mo` files:
+
+```bash
+$ python setup.py compile_catalog -t mkdocs -l es
+running compile_catalog
+18 of 18 messages (100%) translated in mkdocs/themes/mkdocs/locales/es/LC_MESSAGES/messages.po
+compiling catalog mkdocs/themes/mkdocs/locales/es/LC_MESSAGES/messages.po to mkdocs/themes/mkdocs/locales/es/LC_MESSAGES/messages.mo
+
+$ python setup.py compile_catalog -t readthedocs -l es
+running compile_catalog
+12 of 12 messages (100%) translated in mkdocs/themes/readthedocs/locales/es/LC_MESSAGES/messages.po
+compiling catalog mkdocs/themes/readthedocs/locales/es/LC_MESSAGES/messages.po to mkdocs/themes/readthedocs/locales/es/LC_MESSAGES/messages.mo
+```
+
+You will get the following file structures:
+
+```text
+mkdocs/themes/mkdocs/locales
+├── es
+│   └── LC_MESSAGES
+│       ├── messages.mo
+│       └── messages.po
+...
+mkdocs/themes/readthedocs/locales
+├── es
+│   └── LC_MESSAGES
+│       ├── messages.mo
+│       └── messages.po
+```
+
+Then just modify the `mkdocs.yml` file at the root of the project to add your
+development locale to test:
+
+```yaml
+theme:
+    name: mkdocs
+    locale: es
+```
+
+And run `mkdocs serve` to check out your new localized version of the theme.
+
+!!! Note
+    The build and release process takes care of compiling and distributing
+    all locales to end users so you only have to worry about contributing the
+    actual text translation `messages.po` files (the rest is git ignored).
```

Comment:
This is correct. However, `build` in this instance is referring to building the Python package, not building the site using MkDocs. We need to reword to make this clear.